nautobot/extras/models/relationships.py

```diff
@@ -233,6 +235,17 @@ def get_for_model(self, model):
             self.get_queryset().filter(destination_type=content_type),
         )
 
+    def get_required_for_model(self, model):
+        """
+        Return all required Relationships assigned to the given model.
+        """
+        required_types = [req_type for req_type in RelationshipSidesRequiredChoices.values() if req_type]
+        source, destination = self.get_for_model(model)
+        return {
+            "source": destination.filter(required_side__in=required_types),
+            "destination": source.filter(required_side__in=required_types),
```

Comment:
This seems incorrect/confusing to me:

1) it inverts the meaning of `source` and `destination` as compared to `get_for_model`?
2) why `required_side__in=required_types` in both cases, not just `required_side="source"` on one side and `required_side="destination"` on the other?